### PR TITLE
Enhancements to table printing and MNIST scripts

### PR DESCRIPTION
**Pull request by subutai**
Enhancements to table printing scripts, and MNIST scripts to make them more consistent with GSC scripts.

### CODE REVIEW

File: projects/whydense/gsc/exps.cfg [hunk 1]
```diff
@@ -628,52 +631,102 @@ dropout = 0.0
...
 batch_size = 16
 
 
+[denseCNN2]
+linear_n = (1000,)
+linear_percent_on = (1.0,)
+cnn_out_channels = (64, 64)
+cnn_percent_on = (1.0, 1.0)
+cnn_weight_sparsity = (1.0, 1.0)
+k_inference_factor = 1.5
```
Comment (lscheinkman) [inline projects/whydense/gsc/exps.cfg]: `k_inference_factor`  is not used. it should be removed

File: projects/whydense/gsc/run.py [hunk 2]
```diff
@@ -222,6 +222,8 @@ def train(config, experiments, num_cpus, num_gpus, redis_address, show_list):
             reuse_actors=config.get("reuse_actors", False),
             trial_executor=config.get("trial_executor", None),
             raise_on_failed_trial=config.get("raise_on_failed_trial", True),
+            keep_checkpoints_num=1,
+            checkpoint_score_attr="mean_accuracy",
```
Comment (lscheinkman) [inline projects/whydense/gsc/run.py]: Nice!

File: projects/whydense/gsc/run.py [hunk 3]
```diff
@@ -306,5 +308,5 @@ def noise(config, experiments, num_cpus, num_gpus, redis_address):
 
 
 if __name__ == "__main__":
-    set_random_seed(18)
+    set_random_seed(20)
```
Comment (lscheinkman) [inline projects/whydense/gsc/run.py]: what's wrong with '18'? 😄

File: projects/whydense/mnist/exps.cfg [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
+no_cuda = False
+log_interval = 2000
+
+# Common network parameters
+weight_sparsity = (0.3, )
+cnn_weight_sparsity = (1.0,)
+use_batch_norm = False
+boost_strength = 1.5
+boost_strength_factor = 0.85
+k_inference_factor = 1.5
```
Comment (lscheinkman) [inline projects/whydense/mnist/exps.cfg]: Maybe the default value should be `1.0` here

File: projects/whydense/gsc/exps.cfg (hunk 1 above)
Comment (subutai) [reply projects/whydense/gsc/exps.cfg]: I switched the default to 1.0, but kept the old experiments at 1.5

File: nupic/research/support/__init__.py [hunk 5]
```diff
@@ -18,4 +18,4 @@
 #  http://numenta.org/licenses/
 #
 from .parse_config import parse_config
-from .ray_utils import load_ray_tune_experiment
+from .ray_utils import load_ray_tune_experiment, load_ray_tune_experiments
```
[the remark this answers is not in the document]
Comment (subutai) [reply nupic/research/support/__init__.py]: load_ray_tune_experiment is still used in the CIFAR code, so maybe I'll take care of this once I redo the CIFAR stuff.

File: projects/whydense/test_score_table.py [hunk 6]
```diff
@@ -54,21 +54,30 @@
...
+                               "Session"]]
 
     # Load and parse experiment configurations
     configs = parse_config(config, experiments, globals_param=globals())
 
+    # Use the appropriate plus/minus sign for latex
+    if tablefmt == "grid":
+        pm = "±"
+    else:
+        pm = "$\\pm$"
```
Comment (lscheinkman) [inline projects/whydense/test_score_table.py]: This value depends on `tablefmt`.

Comment (subutai) [reply projects/whydense/test_score_table.py]: True, but there are only two options (enforced by click).

**Pull request merged**